distribution/docs/src/main/jdocs/content/_application-references/managing-broker-app-contents.adoc

```diff
@@ -0,0 +1,279 @@
+	"timestamp": 1485971158,
+	"status": 200
+}
+----
+
+===== Securing the ${message-broker} Application
+
+${branding} can be configured to use Artemis to perform authentication and authorization against an LDAP server.
+
+Artemis provides the ability to apply role-based security to queues based on addresses
+(see https://activemq.apache.org/artemis/docs/${artemis.version}/security.html[the Artemis documentation]
+for details).
+It can be configured to use an LDAP server to perform authentication and authorization for users who connect to it.
+
+[IMPORTANT]
+====
+If you are setting up multiple ${branding} instances in a cluster for high availability, then you will need to perform these steps on each instance.
+====
+
+The ${ddf-security} STS LDAP Login and ${ddf-security} STS LDAP Claims Handler bundles are responsible for authenticating and authorizing users with your LDAP server.
+To configure them for your LDAP server, follow the instructions in <<${ddf-security} STS LDAP Login,${ddf-security} STS LDAP Login>> and <<${ddf-security} STS LDAP Claims Handler,${ddf-security} STS LDAP Claims Handler>>.
+
+Once the STS LDAP Login and Claims Handlers are configured, update `<${branding}_HOME>/etc/org.apache.activemq.artemis.cfg` to use the `ldap` realm (just change `domain=karaf` to `domain=ldap`):
+
+.<${branding}_HOME>/etc/org.apachc.activemq.artemis.cfg
+----
+domain=ldap
+----
+
+${branding} uses two roles in the security settings for Artemis: `manager` and `broker-client`.
+
+.<${branding}_HOME>/etc/artemis.xml
+[source,xml]
+----
+<security-setting match="#">
+    <permission type="createNonDurableQueue" roles="manager,broker-client"/>
+    <permission type="deleteNonDurableQueue" roles="manager,broker-client"/>
+    <permission type="createDurableQueue" roles="manager"/>
+    <permission type="deleteDurableQueue" roles="manager"/>
+    <permission type="consume" roles="manager,broker-client"/>
+    <permission type="browse" roles="manager,broker-client"/>
+    <permission type="send" roles="manager,broker-client"/>
+    <permission type="manage" roles="manager"/>
+</security-setting>
+----
+
+Users with the role `manager` have full permissions, but users with the role `broker-client` cannot
+create or delete durable queues or invoke management operations.
+
+Your LDAP should have groups that correspond to these roles so that members of those groups will have
+the correct permissions when connecting to Artemis to send or consume messages.
+Alternatively, you can choose roles other than `manager` and `broker-client`, which may be useful if your LDAP already
+has groups that you would like to use as Artemis roles.
+If you wish to use different roles, just replace `manager` and/or `broker-client` in the `<security-setting>` in `artemis.xml` with the roles you would like to use.
+
+At this point, an `ldap` realm has been added and Artemis is using it for authentication and authorization,
```

Comment:
Looks like can cut some of this out we don't need this paragraph down to `Artemis Broker Connection Configuration` since it is covered in more detail elsewhere.